lib/widgets/lightbox.dart

```diff
@@ -234,17 +252,53 @@ class _ImageLightboxPageState extends State<_ImageLightboxPage> {
 
   @override
   Widget build(BuildContext context) {
+    final thumbnailUrl = widget.thumbnailUrl;
     return _LightboxPageLayout(
       routeEntranceAnimation: widget.routeEntranceAnimation,
       message: widget.message,
+      buildAppBarFooter: _loadingProgress == null ? null : _buildAppBarFooter,
```

Comment:
It doesn't cause the layout shift, because `_LightboxPageLayout` sets [`Scaffold.extendBodyBehindAppBar`](https://api.flutter.dev/flutter/material/Scaffold/extendBodyBehindAppBar.html) to `true` [here](https://github.com/zulip/zulip-flutter/blob/7285325a3068b40e1feed9db4909aef04d31946f/lib/widgets/lightbox.dart#L189).

Reply:
Ah right, that makes sense, thanks.